src/extensibility/ExtensionManagerView.js

```diff
@@ -250,6 +250,12 @@ define(function (require, exports, module) {
                             info.metadata[prop] = info.metadata["package-i18n"][locale][prop];
                         }
                     });
+                    if (info.metadata["package-i18n"][locale].hasOwnProperty("author")) {
+                        info.metadata.author = info.metadata["package-i18n"][locale].author;
+                    }
+                    if (info.metadata["package-i18n"][locale].hasOwnProperty("contributors")) {
+                        info.metadata.contributors = info.metadata["package-i18n"][locale].contributors;
+                    }
```

Comment:
I'm not sure I understand why these need special treatment, instead of just adding `"author"` and `"contributors"` to the array above.